<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="simulation">
       <attribute name="title">
        <string>Simulation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>application</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLineEdit" name="date"/>
          </item>
          <item row="1" column="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>version</string>
              </property>
              <property name="buddy">
               <cstring>application_version</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>name</string>
              </property>
              <property name="buddy">
               <cstring>application_name</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>constraint</string>
              </property>
              <property name="buddy">
               <cstring>application_constraint</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="application_name">
              <item>
               <property name="text">
                <string>gPROMS</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="application_version">
              <property name="text">
               <string>4.0.0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="application_constraint">
              <item>
               <property name="text">
                <string>AT-LEAST</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>date</string>
            </property>
            <property name="buddy">
             <cstring>date</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="filetype">
            <item>
             <property name="text">
              <string>sinterconfig</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>filetype-version</string>
            </property>
            <property name="buddy">
             <cstring>filetype_version</cstring>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>config-version</string>
            </property>
            <property name="buddy">
             <cstring>config_version</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLineEdit" name="title"/>
          </item>
          <item row="4" column="2">
           <widget class="QComboBox" name="filetype_version">
            <item>
             <property name="text">
              <string>0.3</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>author</string>
            </property>
            <property name="buddy">
             <cstring>author</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>filetype</string>
            </property>
            <property name="buddy">
             <cstring>filetype</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLineEdit" name="description"/>
          </item>
          <item row="9" column="2">
           <widget class="QLineEdit" name="config_version">
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>model</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>description</string>
            </property>
            <property name="buddy">
             <cstring>description</cstring>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="2">
             <widget class="QPushButton" name="digest_calculate">
              <property name="text">
               <string>Calculate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>file</string>
              </property>
              <property name="buddy">
               <cstring>model_file</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="model_file"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="model_digest_value"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>DigestValue</string>
              </property>
              <property name="buddy">
               <cstring>model_digest_value</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>SignatureMethodAlgorithm</string>
              </property>
              <property name="buddy">
               <cstring>mode_digest_method</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="mode_digest_method">
              <item>
               <property name="text">
                <string>sha1</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="model_browse">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="2">
           <widget class="QLineEdit" name="author"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>title</string>
            </property>
            <property name="buddy">
             <cstring>title</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>384</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="set_working_dir">
          <property name="text">
           <string>Set Working Directory</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="input_files">
       <attribute name="title">
        <string>Input Files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="add_input_file">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="del_input_file">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="input_files_table">
          <column>
           <property name="text">
            <string>file</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>DigestValue</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SignatureMethodAlgorithm</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="settings_table">
          <row>
           <property name="text">
            <string>ProcessName</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>password</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>default</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>description</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>string</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string/>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="0" column="2">
           <property name="text">
            <string>Simulation specific setting: ProcessName</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>string</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string/>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item row="1" column="2">
           <property name="text">
            <string>Simulation specific setting: Password</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="input">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="add_input">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_input">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="inputs_table">
          <column>
           <property name="text">
            <string>name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>default</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>max</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>min</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>units</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="output">
       <attribute name="title">
        <string>Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="add_output">
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delete_output">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="outputs_table">
          <column>
           <property name="text">
            <string>name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>path</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>default</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>units</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuLoad"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>application_name</tabstop>
  <tabstop>application_version</tabstop>
  <tabstop>application_constraint</tabstop>
  <tabstop>filetype</tabstop>
  <tabstop>filetype_version</tabstop>
  <tabstop>title</tabstop>
  <tabstop>author</tabstop>
  <tabstop>date</tabstop>
  <tabstop>description</tabstop>
  <tabstop>config_version</tabstop>
  <tabstop>model_file</tabstop>
  <tabstop>digest_calculate</tabstop>
  <tabstop>model_digest_value</tabstop>
  <tabstop>mode_digest_method</tabstop>
  <tabstop>outputs_table</tabstop>
  <tabstop>delete_output</tabstop>
  <tabstop>add_output</tabstop>
  <tabstop>inputs_table</tabstop>
  <tabstop>delete_input</tabstop>
  <tabstop>add_input</tabstop>
  <tabstop>settings_table</tabstop>
  <tabstop>add_input_file</tabstop>
  <tabstop>del_input_file</tabstop>
  <tabstop>input_files_table</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
